docs/lineage/gravitino-server-lineage.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Gravitino server Lineage support"
+slug: /lineage/gravitino-server-lineage
+keyword: Gravitino OpenLineage
+license: "This software is licensed under the Apache License version 2."
+---
+
+## Overview
+
+Gravitino server provides a pluginable lineage framework to receive, process, and sink OpenLineage events. By leveraging this, you could do custom process for the lineage event and sink to your dedicated systems.
+
+## Lineage Configuration
+
+| Configuration item                            | Description                                                                                                                                                                                           | Default value                                          | Required | Since Version |
+|-----------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------|----------|---------------|
+| `gravitino.lineage.source`                    | The name of lineage event source. The default `http` event source will .                                                                                                                              | http                                                   | No       | 0.9.0         |
+| `gravitino.lineage.${sourceName}.sourceClass` | The name of the lineage source class which should implement `org.apache.gravitino.lineage.source.LineageSource` interface.                                                                            | (none)                                                 | No       | 0.9.0         |
+| `gravitino.lineage.processorClass`            | The name of the lineage processor class which should implement `org.apache.gravitino.lineage.processor.LineageProcessor` interface. The default noop processor will do nothing about the run event.   | `org.apache.gravitino.lineage.processor.NoopProcessor` | No       | 0.9.0         |
+| `gravitino.lineage.sinks`                     | The name of lineage event sinks.                                                                                                                                                                      | log                                                    | No       | 0.9.0         |
```

Comment:
Is the value a plural? say, comma-separated strings?

Reply:
yes, updated